### PR TITLE
Add media twig extensions

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/168
https://github.com/open-orchestra/open-orchestra-docs/pull/167

### CODE REVIEW

File: en/developer_guide/twig_extensions.rst [hunk 1]
```diff
@@ -3,6 +3,17 @@ Twig extensions
 
 Some twig extensions have been developed specifically for Open Orchestra and are available for use.
 
+Media functions
+---------------
+
+If you are using the media bundles you will probably have to manipulate media within your twig templates. You
```
Comment (veolys-nanne) [inline en/developer_guide/twig_extensions.rst]: coma between bundles and you

**Comment by veolys-nanne** [comment]
:+1:

File: en/developer_guide/twig_extensions.rst [hunk 2]
```diff
@@ -3,6 +3,17 @@ Twig extensions
...
+can use these twig extensions to help you:
+
+* display_media(mediaId, format): render a media (or an alternative) using the DisplayMediaManager
+* get_media_url(mediaId, format): get the url of a media (or an alternative)
```
Comment (alavieille) [inline en/developer_guide/twig_extensions.rst]: use `display_media(mediaId, format)` same for others

**Comment by ngilain** [comment]
@itkg-nanne , @alavieille : update done

**Comment by alavieille** [comment]
:+1:

**Pull request merged**